Dockerfile

```diff
@@ -14,6 +14,9 @@ WORKDIR /app
 
 EXPOSE 3000
 
+# install dependencies inside the image for prod
```

Comment:
I assume this is the case but then, I see that the worker does never run `npm ci`.

I think this means that every app container includes the worker. So we don't have 4 app services and 4 worker services running, but 4 app+worker services.

Reply:
ahh, ohh